rts/Lua/LuaMathExtra.cpp

```diff
@@ -184,6 +185,34 @@ int LuaMathExtra::smoothstep(lua_State* L) {
 	return 1;
 }
 
+
+/*** Returns the normalize vector of an given vector.
+ *
+ * @function math.normalize
+ * @number x
+ * @number y
+ * @number z
+ * @treturn nil|normalize vector
```

Comment:
Accepts and returns N numbers. Check out the docs for `math.diag` for an example.